charts/kube-otel-stack/values.yaml

```diff
@@ -154,12 +166,24 @@ metricsCollector:
         detectors: [env]
         timeout: 2s
         override: false
+      k8sattributes:
+        passthrough: false
+        extract:
```

Comment:
From the docs 
```
If Pod association rules are not configured, resources are associated with metadata only by connection's IP Address.
```

Reply:
So I think we can set a pod association rule to find metadata associated with other resource attributes. I tried this with the `k8s.pod.name` instead of the defaulting `k8s.pod.ip` by adding
```
       pod_association:
          - sources:
              - from: resource_attribute
                name: k8s.pod.name
       extract:
          metadata:
            - k8s.cluster.name
            - k8s.namespace.name
            - k8s.pod.name
            - k8s.pod.uid
            - k8s.node.name
            - k8s.pod.start_time
            - container.image.tag
            - container.image.name

```

Reply:
This allowed me to extract some extra metadata labels like `k8s.pod.start_time` , `container.image.tag`, `container.image.name`. Not sure why these don't show up when using the defaulting pod.ip association. Here is a notebook showing these extra labels populating once I add this block to my processor config: https://app.lightstep.com/dev-mohosman/notebooks/hn2FDQpW

Reply:
I've only tested this for metrics with statefulset using prom receiver. The `container.*` labels are dependent on `k8s.container.name` being set (not sure what sets this maybe prometheus?), so these might disappear if not using the prometheus receiver